src/components/Home/Ritika/Login/Login.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { LoginWrapper } from './Login.styled';
+import { toast } from 'react-toastify';
+import { validatePassword, validateUser } from '../Week3/utils/validation';
+import message from '../Week2/utils/constants/constants.json';
+
+const Login: React.FC = () => {
+  const navigate = useNavigate();
+  const [userName, setUserName] = useState<string>('');
+  const [userPassword, setUserPassword] = useState<string>('');
+
+  const handleUserNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setUserName(e.target.value);
+  };
+  const handleUserPasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setUserPassword(e.target.value);
+  };
+
+  const handleSubmit = () => {
+    if (!validateUser(userName, userPassword)) return;
```

Comment:
Here, You are validating user first and then validation password.
It should be reverse while validating